src/styles/homepage.css

```diff
@@ -0,0 +1,74 @@
+.shows {
+  width: 100%;
+  display: grid;
+  grid-template-columns: auto auto auto;
+  grid-gap: 45px 30px;
+  padding: 20px 0;
+  background: #242222;
```

Comment:
[OPTIONAL} Kindly you can add more columns

Reply:
Just like you had suggested earlier, kindly lets do the styling after we add all the functions

Reply:
you are right, so let's do it later